bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableClient.scala

```diff
@@ -232,10 +280,20 @@ trait VoTableClient {
 }
 
 object VoTableClient extends VoTableClient {
+
+  def defaultBackend(n: CatalogName): VoTableBackend =
+    n match {
+      case CatalogName.Gaia   => GaiaBackend
+      case CatalogName.SIMBAD => SimbadNameBackend
+      case _                  => ConeSearchBackend
+    }
+
   /**
    * Do a query for targets, it returns a list of targets and possible problems found
    */
-  def catalog(query: CatalogQuery, backend: VoTableBackend = ConeSearchBackend)(ec: ExecutionContext): Future[QueryResult] = {
+  def catalog(query: CatalogQuery, explicitBackend: Option[VoTableBackend])(ec: ExecutionContext): Future[QueryResult] = {
+    val backend = explicitBackend.getOrElse(defaultBackend(query.catalog))
```

Comment:
I changed the `catalog` to make the "backend" default according to the catalog instead of always being `ConeSearchBackend`.